chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/surveillance/report/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * 
```

Comment:
There are a couple of these package-info files with duplicate /* */ blocks. Didn't mention it on the first one, but with more than one I felt the need to call it out